language-command.php

```diff
@@ -41,3 +29,9 @@
 if ( class_exists( 'WP_CLI\Dispatcher\CommandNamespace' ) ) {
 	WP_CLI::add_command( 'language', 'Language_Namespace' );
 }
+
+function wpcli_language_check_requirements() {
```

Comment:
This will break in some cases due to the way Composer  `"files"` works (ie. it will require, not require_once).

Turn it into an actual closure to avoid this:
```suggestion
$wpcli_language_check_requirements = function () {
```